application/backend/eye_tracker_newsdk.py

```diff
@@ -66,41 +63,17 @@ def __init__(self):
         self.last_pupil_right = -1
         self.LastTimestamp = -1
         self.init_emdat_global_features()
+
+        # Choose which eye tracker to use here: you can define eye_tracker to be any eye tracker class implemented inside the file eye_tracker_class.py
+        self.eye_tracker = SimulationEyeTracker()
```

Comment:
I wonder if we can avoid that here, by having a dictionary somewhere with the mapping from the eye-tracker names (from params.py) to their corresponding class, like in eye_tracker_class. This is to make all files generic except for eye_tracker_class.py and params.py.
Something like (in : EyeTracker class)
eye_tracker_set = { "simulation": "SimulationEyeTracker", "Tobii T120": "TobiiT120EyeTracker", "IS4_Large_Peripheral": "Tobii4CEyeTracker"}

So that here we can do: 
eye_tracker_class_resolver = EyeTracker().eye_tracker_set[params.EYETRACKER_TYPE]
self.eye_tracker = getattr(application.backend.eye_tracker_class, eye_tracker_class_resolver)()

does it sound feasible?